package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "./node_modules/tape/bin/tape ./tests/unit/*.js",
+    "integration-tests": "./node_modules/tape/bin/tape ./tests/integration/*.js",
```

Comment:
It should, but once it's configured to use an actual account, these integration tests require a valid project